### DISCUSSION TITLE
see your prompt when you mouse over jpg in widows explorer

### DISCUSSION THREAD

**Discussion by chuckkay** [vladmandic/sdnext#1535] [Show and tell]
Thanks to Vlad for making the default filetype .jpg as it has legit exif information now..
so here's a Cool Trick  the tooltip feature in windows explorer shows file size and dimension when you hover with your mouse over a thumbnail... would you like to see the prompt too?   I bet you would especially if you were generating a 100's of images with an xyz plot or prompt macro.. here's how.

in regedit section of [HKEY_CLASSES_ROOT\SystemFileAssociations.jpg]
go to infoTip section and change 'System.Keywords' to 'System.Comment' 
that's it!!!!  and then hover over jpg files in windows explorer and get This
![Screenshot 2023-06-30 125008](https://github.com/vladmandic/automatic/assets/10617746/9899f0de-4f13-4275-b3e3-a31fc9faf2b9)

**Comment by tkiikt** [comment]
you might like this viewer: [Breadboard](https://github.com/cocktailpeanut/breadboard)

**Comment by vladmandic** [comment]
@chuckkay oh, nice catch!

actually, why not populate keywords to start with, so no registry change is needed.
but placing entire infotext is a lot - lets come up with a short string that would still be useful?
maybe i can re-use filename template generator for that so user can specify a template to go into exif keywords? hmm...

**Reply by chuckkay** [reply]
Would love that, 
So tags section for comma separated prompts.. And maybe steal some other unused exif fields too and use them. like Instead of iso & shutter speed could contain things like steps and sampler.  And

but i would still keep comments section unchanged  as is for a full list of prompts settings etc…

**Comment by Pedroman1** [comment]
you guys are amazing love this stuff
